app/views/comments/_comment.html.erb

```diff
@@ -6,9 +6,10 @@
   <div class = 'comment-footer'>
     posted by: <%= comment.commenter %>
 
-    <%= link_to article_comments_path(comment),
+    <%= link_to article_comment_path(comment.article, comment),

```

Comment:
You should just be able to pass the comment (or even better, the comment ID) - though you'll need to change your controller code.

Reply:
I've tried just passing one or the other but haven't had any success

(see "8 Deleting comments" in https://edgeguides.rubyonrails.org/getting_started.html )

Reply:
Ah okay, both must be necessary to construct the `link_to`, since we're not using a `path` helper. Sorry, I didn't think of that before! My bad 😞

And you will want the article number, for the redirect. 

So you had it right to begin with, I'm sorry for steering you wrong. 😿